tests/bugs/gh-463.slang.expected

```diff
@@ -1,6 +1,8 @@
 result code = -1
 standard error = {
 tests/bugs/gh-463.slang(7): error 30015: undefined identifier 'COUNT'.
+(0): note 31122: see defintion for attribute parameter 'instance.count'
```

Comment:
I don't think this note is useful in this case. It raises more questions than it answers. I might advocate for removing all the code that is emitting these notes unless they are in cases where the preceding diagnostic specifically refers to something about the parameter.

Reply:
Fixed.